.github/workflows/tag.yaml

```diff
@@ -0,0 +1,22 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github_token: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
+          tag_prefix: ""
+      - name: Create a GitHub release
```

Comment:
Why do we need to create a GitHub release for each new tag? (who will read them?)

Reply:
Good point. I just copied from ../automation-flavors without :brain:. That probably does not need this either.

Reply:
deleted.